HomeworkLessonFour091122/LessonFourTaskSix.java

```diff
@@ -0,0 +1,18 @@
+// napisat' class kotoruy vuvodit v console massiv chisel v obratnom poryadke, polychennuy iz chisla vvedenogo v console
+
+import java.util.Scanner;
+
+class LessonFourTaskSix {
+    public static void main(String[] args) {
+        Scanner myScan = new Scanner(System.in);
+        System.out.println("Enter the number:");
+        int num = myScan.nextInt();
```

Comment:
add if statement if entered value is less than 1

Reply:
fixed